pkg/backup/actions/csi/pvc_action.go

```diff
@@ -340,6 +342,13 @@ func (p *pvcBackupItemAction) Execute(
 			dataUploadLog.Info("DataUpload is submitted successfully.")
 		}
 	} else {
+		p.log.Debugf("Patching PVC request size to fit the volumesnapshot restore size %d", vsc.Status.RestoreSize)
```

Comment:
Are we sure we have vsc.Status.RestoreSize always available at this point? If not we will get nil crash.

Reply:
added wait for restore size to:
WaitUntilVSCHandleIsReady